packages/fxa-auth-server/CONTRIBUTING.md

```diff
@@ -7,66 +7,73 @@ mailing list or through issues here on GitHub.
 - and of course, [the issues list](https://github.com/mozilla/fxa-auth-server/issues)
 
-## Bug Reports ##
+## Bug Reports
 
 You can file issues here on GitHub. Please try to include as much information as you can and under what conditions
 you saw the issue.
 
-## Sending Pull Requests ##
+## Sending Pull Requests
 
 Patches should be submitted as pull requests (PR).
 
 Before submitting a PR:
+
 - Your code must run and pass all the automated tests before you submit your PR for review. "Work in progress" pull requests are allowed to be submitted, but should be clearly labeled as such and should not be merged until all tests pass and the code has been reviewed.
   - Run `grunt eslint` to make sure your code passes linting.
   - Run `npm test` to make sure all tests still pass.
 - Your patch should include new tests that cover your changes. It is your and your reviewer's responsibility to ensure your patch includes adequate tests.
 
 When submitting a PR:
+
 - You agree to license your code under the project's open source license ([MPL 2.0](/LICENSE)).
 - Base your branch off the current `master` (see below for an example workflow).
 - Add both your code and new tests if relevant.
 - Run `grunt eslint` and `npm test` to make sure your code passes linting and tests.
 - Please do not include merge commits in pull requests; include only commits with the new relevant code.
 - Your commit message must follow the
-[commit guidelines](https://github.com/mozilla/fxa/blob/master/CONTRIBUTING.md#git-commit-guidelines).
+  [commit guidelines](https://github.com/mozilla/fxa/blob/master/CONTRIBUTING.md#git-commit-guidelines).
 
 After your PR is merged:
+
 - Add yourself to the [AUTHORS](/AUTHORS) file so we can publicly recognize your contribution.
 
 See the main [README.md](/README.md) for information on prerequisites, installing, running and testing.
 
-## Code Review ##
+## Code Review
 
 This project is production Mozilla code and subject to our [engineering practices and quality standards](https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Committing_Rules_and_Responsibilities). Every patch must be peer reviewed. This project is part of the [Firefox Accounts module](https://wiki.mozilla.org/Modules/Other#Firefox_Accounts), and your patch must be reviewed by one of the listed module owners or peers.
 
-## Example Workflow ##
+## Example Workflow
 
 This is an example workflow to make it easier to submit Pull Requests. Imagine your username is `user1`:
 
 1. Fork this repository via the GitHub interface
 
 2. The clone the upstream (as origin) and add your own repo as a remote:
 
-    ```sh
-    $ git clone https://github.com/mozilla/fxa-auth-server.git
-    $ cd fxa-auth-server
-    $ git remote add user1 git@github.com:user1/fxa-auth-server.git
-```
+   ```sh
+   $ git clone https://github.com/mozilla/fxa-auth-server.git
+   $ cd fxa-auth-server
+   $ git remote add user1 git@github.com:user1/fxa-auth-server.git
+   ```
+
+````
```

Comment:
This is weird, it seems to have got confused by the mismatched indentation of the backticks below and decided it needs to insert 4 of them here instead. Presumably if we indent the bottom one it will do the right thing here.

Reply:
Might be worth grepping for 4 backticks together before committing this I guess, just in case there's any more.

Reply:
there's a few but i don't want to redo all the commits to fix them in the right ones so i'll fix it at the end

Reply:
i just noticed these were already borked before formatting ;)